zokrates_js/tests/tests.js

```diff
@@ -0,0 +1,122 @@
+const assert = require('assert');
+const { initialize } = require('../node/index.js');
+
+describe('tests', function() {
+
+    before(function (done) {
```

Comment:
Just to clarify: this initializes a single time before the suite of test, it's fine because `zokrates` is stateless, otherwise we would want `beforeEach`? If it's the case, would be nice to add a comment, or use `beforeEach`

Reply:
`before` is appropriate for this case, as we only need to initialize it once before running the tests